### PR TITLE
Rework the TOC

### PR DESCRIPTION
**Pull request by dunglas**
* Merge TOC of all components (to create a global index)
* Fixed broken links
* Switch to Title Case for titles

### CODE REVIEW

File: tutorial/index.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ to improve and professionalize the PHP ecosystem. The PHP world has closed the g
 more innovative than them.
 
 But in critical area I've described previously, many things can be improved. Almost all existing solutions are still [designed
-and documented](https://symfony.com/doc/2.7/book/page_creation.html) to create websites the old way: a server generate then
+and documented](https://symfony.com/doc/current/book/page_creation.html) to create websites the old way: a server generate then
```
Comment (teohhanhui) [inline tutorial/index.md]: generate -> generates

send -> sends

File: tutorial/index.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ to improve and professionalize the PHP ecosystem. The PHP world has closed the g
...
 send plain-old HTML documents to browsers.
 
-[API Platform](https://api-platform.com) is a brand new set of tools to kickstart modern web projects. It's a framework for
-API-first projects built on top of Symfony. Like other modern frameworks such as Zend Framework and Symfony, it's both a
+[API Platform](https://api-platform.com) is a new generation set of tools to kickstart modern web projects. It is a framework
```
Comment (teohhanhui) [inline tutorial/index.md]: a set of modern tools?

File: tutorial/index.md [hunk 3]
```diff
@@ -16,19 +16,21 @@ to improve and professionalize the PHP ecosystem. The PHP world has closed the g
...
 But in critical area I've described previously, many things can be improved. Almost all existing solutions are still [designed
-and documented](https://symfony.com/doc/2.7/book/page_creation.html) to create websites the old way: a server generate then
-send plain-old HTML documents to browsers.
+and documented](https://symfony.com/doc/current/book/page_creation.html) to create websites the old way: a server generates
+then sends plain-old HTML documents to browsers.
 
-[API Platform](https://api-platform.com) is a brand new set of tools to kickstart modern web projects. It's a framework for
-API-first projects built on top of Symfony. Like other modern frameworks such as Zend Framework and Symfony, it's both a
-full-stack all-in-one framework and a set of independent PHP components and bundles that can be used separately.
+[API Platform](https://api-platform.com) is a new generation set of modern tools to kickstart web projects. It is a framework
```
Comment (teohhanhui) [inline tutorial/index.md]: Oops... Sorry, I wasn't paying attention to the second half of the sentence.

Let me rephrase it:

API Platform is a set of tools for building modern web projects.

**Comment by soyuka** [comment]
:+1:
